pkg/clusterautoscaler/flags.go

```diff
@@ -152,30 +157,31 @@ func AddClusterAutoscalerFlags(cmd *cobra.Command, prefix string) *AutoscalerArg
 		&args.BalancingIgnoredLabels,
 		fmt.Sprintf("%s%s", prefix, balancingIgnoredLabelsFlag),
 		nil,
-		"A comma-separated list of label keys that cluster autoscaler should ignore when considering node group similarity.",
+		fmt.Sprintf("A comma-separated list of label keys that cluster autoscaler should ignore when "+
+			"considering node group similarity. %s", classicOnlyHelpMsg),
 	)
 
 	// Resource Limits
 
 	cmd.Flags().IntVar(
 		&args.ResourceLimits.MaxNodesTotal,
 		fmt.Sprintf("%s%s", prefix, maxNodesTotalFlag),
-		0,
+		249,
```

Comment:
seems we should keep the default value as 0 due to discussion here: https://redhat-internal.slack.com/archives/CB53T9ZHQ/p1738709990158539?